File: Javascript/1051-1100/1051. Height Checker.js
var heightChecker = function(heights) {
        let expected = heights.slice().sort((a, b) => a - b);

        let difference  = 0;

        for(let index=0; index<heights.length; index++)
            if(heights[index] != expected[index]) difference++;

        return difference;
};

File: Java/1051-1100/1051. Height Checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] expected = heights.clone();
        Arrays.sort(expected);

        int difference  = 0;

        for(int index=0; index<heights.length; index++)
            if(heights[index] != expected[index]) difference++;

        return difference;
    }
}
